osx/setup.sh

```diff
@@ -240,6 +243,27 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+# Install requirements for sphinx
+# Reference ulimit: https://github.com/substack/node-browserify/issues/431
+echo "$STEP/$STEPS. Running npm install... on '$KA_LITE_DIR' "
+$PYRUN_PIP install -r "$KA_LITE_DIR/requirements_sphinx.txt"
+cd $KA_LITE_DIR
+if [ -d "$KA_LITE_DIR" ]; then
+    echo "Install npm.."
+    if ! command -v npm > /dev/null; then
+        echo "Abort npm is not installed."
+        exit 1
+    else
+        npm install
+        ulimit -n 2560
+        node build.js
+        cd $KA_LITE_DIR/docs
+        $PYRUN_SPHINX_BUILD -b html -d _build/doctrees   . _build/html
```

Comment:
So we build the docs locally. I don't think this will contain the screenshots though. Maybe you can use @MCGallaspy's zip from field? http://field.learningequality.org/docs/0.14/docs.tar.gz